misc/lint.sh

```diff
@@ -41,6 +41,7 @@ lint_html () {
   html5validator --root "$SITE_DIR" --show-warnings --ignore-re \
     'Illegal character in query: "\|" is not allowed' \
     '"(autocorrect|autocapitalize)" not allowed on element "input"' \
```

Comment:
> '"(autocorrect|autocapitalize)" not allowed on element "input"' 
is not allowed

Reply:
This is a third-party error message, so I cannot change it :disappointed_relieved: